content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/sendcloud_shipping.rst

```diff
@@ -217,6 +217,34 @@ Additionally, the tracking number is now available.
    When return labels are created, Sendcloud will automatically charge the configured Sendcloud
    account.
 
+Shipping rules
+--------------
+
+Optionally, create shipping rules to automatically generate shipping labels tailored to different
+products' needs. For example, a shipping rule can be created for customers shipping expensive
```

Comment:
```suggestion
product needs. For example, a shipping rule can be created for customers shipping expensive
```
just sounds a bit cleaner this way -- totally optional, though. completely your call. 👍